tests/e2e/standalone/init_negative_test.go

```diff
@@ -0,0 +1,71 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package standalone_test
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestStandaloneInitNegatives(t *testing.T) {
```

Comment:
The problem is that it will increase the e2e test time. The negative scenarios in this file requires Dapr to be uninstalled. So if say list tests contain both tests that require Dapr to be installed and uninstalled, each time we will have to bear that overhead.

Reply:
ping @pravinpushkar